bin/check-dns.rb

```diff
@@ -96,15 +96,39 @@ class DNS < Sensu::Plugin::Check::CLI
          long: '--use-tcp',
          boolean: true
 
+  option :count,
```

Comment:
I think count is appropriate here, as it's not retries, it's the count of the number of DNS requests you want to make. I was copying the convention set by ping/fping in doing this.

I reworded the description a little to try and make it clearer.